specs/auditbeat.spec.yml

```diff
@@ -16,6 +16,8 @@ inputs:
       - logstash
       - redis
     command:
+      restart_monitoring_period: 15s
```

Comment:
Is there a way to do this so we don't have to copy it into every input name? Seems like a good way for somebody to forget to do it.

Reply:
anchors should work